### PR TITLE
Update the manifest template for "Load Balanced Web App"

### LINKED ISSUE #198: Update manifest file template for a LoadBalancedWebApp
## Story
As a ECS CLI v2 user that's not familiar with AWS services,  
I want the generated manifest for an application to not have any service-specific terminology,  
So that I can easily configure my application's infrastructure without ramping up on additional concepts.

## Acceptance Criteria
The manifest file for a Load Balanced Web App generated should be:
```yaml
# For all the available properties, please visit:
# <link to our documentation>/manifests/load-balanced-web-app.md
version: '0.1'
name: {appName}
type: load-balanced-web-app
image:
  build: {dockerFilePath}

# Requests to this path will be forwarded to your application.
http:
  path: '*'
```
I'm aiming to be terse and have a very low barrier of entry.  
The full spec for reference is:
```yaml
version: string 
name: string            # App name
type: load-balanced-web-app

image:
  build: string         # Path to your Dockerfile. Required.
  tag: string           # Changes the tag of the image. Default image name is the app's name. Default tag is "latest".
  port: integer         # Container's port. By default 8080.

cpu: integer            # Number of CPU units for the task. Defaults to 512.
memory: integer         # Amount in MiB used by the task. Defaults to 512.
count: integer          # Number of tasks that should be running in the service. Defaults to 1.

http:                # Add a target group to the environment's load balancer.              
  path: string           # Requests to this path will be forwarded to your service. Required.
  method: string         # Method to restrict requests to the service. Defaults to "*".

public: boolean        # Assign public IP address. Defaults to false.

observability: boolean # Enable AppMesh. Defaults to true.

scaling:               # Optional configuration for scaling your service.
  minCount: integer      # Required. Minimum number of tasks that should be running.
  maxCount: integer      # Required. Maximum number of tasks that should be running.
  
  # At least one of these scaling targets must be provided:
  targetCpu: integer           # Scale based on the average cpu utilization of the tasks in your service.
  targetMemory: integer        # Scale based on the average memory utilization.
  targetRequests: integer      # Scale based on the number of requests completed.
  
  cooldown: # Optional amount of time before another scaling activity starts
    scaleIn: integer     # Block other scale in events for X seconds. Defaults to 60.
    scaleOut: integer    # Block other sclae out events for X seconds. Defaults to 60. 

variables:               # Optional container environment variables for plain strings 
  <name>: <value>        # Values are plain strings
  
secrets:                 # Optional container environment variables for secure strings in SSM.
  <name>:
    from: <value>        # Parameter name in SSM.

environments:                    # Override fields for an Archer environment.
  <name>:
    <fields...>
```

### PR DESCRIPTION
**Pull request by efekarakus**
Resolves #198 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/lb-fargate-service/manifest.yml [hunk 1]
```diff
@@ -1,51 +1,48 @@
-# Your application name will be used in naming your resources
-# like log groups, services, etc.
+# The manifest for the "{{.Name}}" application.
```
Comment (efekarakus) [inline templates/lb-fargate-service/manifest.yml]: @clareliguori Do you think these configurations are sufficient for a "frontend" pattern? Is there anything you'd want to see/change that'd be a common usecase?

File: internal/pkg/manifest/lb_fargate_manifest.go [hunk 2]
```diff
@@ -10,42 +10,81 @@ import (
...
+		},
+		LBFargateConfig: LBFargateConfig{
+			RoutingRule: RoutingRule{
+				Path: "*",
+			},
+			ContainersConfig: ContainersConfig{
+				CPU:    256,
+				Memory: 512,
+				Count:  1,
+			},
```
Comment (sonofachamp) [inline internal/pkg/manifest/lb_fargate_manifest.go]: Should we include default Scaling configuration values?

Comment (efekarakus) [reply internal/pkg/manifest/lb_fargate_manifest.go]: I wasn't sure what's your thought on it? enabling autoscaling by default seemed overkill to me but I don't know :P

Comment (sonofachamp) [reply internal/pkg/manifest/lb_fargate_manifest.go]: We could set and expose values that don't actually scale (min and max of 1) to make it obvious where you'd crank up those values if you wanted to assuming there's no inherent cost in doing so.

Comment (efekarakus) [reply internal/pkg/manifest/lb_fargate_manifest.go]: I'll keep it in the comments for now, we'll most likely tweak things in the upcoming weeks anyway :)

**Pull request merged**